pontoon/base/models.py

```diff
@@ -2106,8 +2106,9 @@ def unchanged(self, locale):
         return Q(
             pk__in=self.get_filtered_entities(
                 locale,
-                Q(Q(string=F('entity__string')) | Q(string=F('entity__string_plural'))),
-                lambda x: x.string == x.entity.string,
+                Q(Q(active=True) &
+                  Q(Q(string=F('entity__string')) | Q(string=F('entity__string_plural')))),
+                lambda x: x.active and x.string == x.entity.string,
```

Comment:
We should also compare the string to `string_plural` (unrelated to this bug), so this line should read:

```python
lambda x: x.active and (
    x.string == x.entity.string or
    x.string == x.entity.string_plural
),
```